calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
```

Comment:
Currently `create_calculator` is a function which generates an object with functions. Would it be better as a class instead? For example:
```
export class Calculator {
  constructor(operator) {
    this.operator = operator;
  }
add_number() {...}
calculate() {...}
}
```
This would make it slightly less messy! 😃